src/ccmain/tesseractclass.cpp

```diff
@@ -508,7 +508,7 @@ Tesseract::Tesseract()
       STRING_MEMBER(page_separator, "\f",
                     "Page separator (default is form feed control character)",
                     this->params()),
-      BOOL_MEMBER(glyph_confidences, false,
+      INT_MEMBER(glyph_confidences, 0,
                   "Allows to include glyph confidences in the hOCR output",
```

Comment:
Noah, could you please add help information here on the valid values for `glyph_confidences`?